solution/backend/regulations/rss_feed.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.syndication.views import Feed
+from django.urls import reverse
+from datetime import datetime
+from regcore.models import Part
+from resources.models import AbstractResource
+
+
+class PartFeed(Feed):
+    def get_feed(self, obj, request):
+        feedgen = super().get_feed(obj, request)
+        feedgen.content_type = 'application/xml'  # New standard
+        return feedgen
+    title = 'Federal Register documents RSS Feed'
+    link = '/latest/feed'
+    description = 'Displays the latest federal register documents'
+
+    def processChildren(self, children, title, part, last_updated):
+        results = []
```

Comment:
If you modify the function you wont need to create an empty list, you would just be working with the results list